functions.Rmd

```diff
@@ -32,35 +32,36 @@ map(xs, ~ mean((.x + 5)^2))
 * Avoid using `\()` except for anonymous functions.
 * Avoid using `\()` in a pipe.
 
-## Long lines
+## Multi-line function defintions
 
-There are two options if the function name and definition can't fit on a single line:
+There are two options if the function name and definition can't fit on a single line. In both cases, each argument goes on its own; the difference is how deep you indent it and where you put `)` and `{`:
 
-*   Function-indent: place each argument on its own line, and indent to match the opening `(` of `function`:
+*   **Single-indent**: indent the argument name with a single indent (i.e. two spaces).
+    The trailing `)` and leading `{` go on a new line.
 
     ```{r, eval = FALSE}
-    long_function_name <- function(a = "a long argument",
-                                   b = "another argument",
-                                   c = "another long argument") {
+    # Good
+    long_function_name <- function(
+      a = "a long argument",
+      b = "another argument",
+      c = "another long argument"
+    ) {
       # As usual code is indented by two spaces.
     }
     ```
 
-*   Double-indent: Place each argument of its own **double** indented line.
+*   **Function-indent**: indent the argument name to match the opening `(` of `function`.
```

Comment:
FWIW we use the term `hanging_indent_style` in {lintr}:

https://lintr.r-lib.org/reference/indentation_linter.html

Reply:
Oh yeah I like that better.